apps/api/src/app/auth/framework/subscriber-route.guard.ts

```diff
@@ -1,35 +0,0 @@
-import { Injectable, CanActivate, ExecutionContext } from '@nestjs/common';
-import { Reflector } from '@nestjs/core';
-import { ISubscriberJwt } from '@novu/shared';
-import jwt from 'jsonwebtoken';
-import { PinoLogger } from '@novu/application-generic';
-
-@Injectable()
-export class SubscriberRouteGuard implements CanActivate {
-  constructor(
-    private readonly reflector: Reflector,
-    private readonly logger: PinoLogger
-  ) {}
-
-  canActivate(context: ExecutionContext): boolean {
-    const subscriberRouteGuard = this.reflector.get<string[]>('subscriberRouteGuard', context.getHandler());
-    if (!subscriberRouteGuard) {
-      return true;
-    }
```

Comment:
This guard will go through only if there is a `@SubscriberRoute()` decorator applied on an endpoint:
```tsx
export const SubscriberRoute = () => SetMetadata('subscriberRouteGuard', true);
```

I can't find such decorator or metadata anywhere.